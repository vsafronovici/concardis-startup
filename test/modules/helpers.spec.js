import {
  createReducer,
  createRequestTypes
} from 'modules/helpers'

describe('helpers', () => {
  describe('createReducer', () => {
    it('should return a proper object', () => {
      expect(createReducer('REQUEST')).toMatchSnapshot()
    })
  })

  describe('createRequestTypes', () => {
    it('should return a proper object', () => {
      expect(createRequestTypes('REQUEST')).toMatchSnapshot()
    })
  })
})
