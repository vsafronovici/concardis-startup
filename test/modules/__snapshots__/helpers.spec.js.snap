// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`helpers createReducer should return a proper object 1`] = `[Function]`;

exports[`helpers createRequestTypes should return a proper object 1`] = `
Object {
  "FAILURE": "REQUEST_FAILURE",
  "REQUEST": "REQUEST_REQUEST",
  "SUCCESS": "REQUEST_SUCCESS",
}
`;
